.babelrc

```diff
@@ -1,6 +1,7 @@
 {
-  "presets": ["react", "env"],
+  "presets": ["react", "env", "stage-2"],
   "plugins": [
-    "transform-object-rest-spread"
+    "transform-object-rest-spread",
+    "transform-class-properties"
   ]
 }
```

Comment:
nitpick: This red icon indicates a new line should be below it. you might want to note that in the future

Reply:
Noted! Thanks.